modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -36,3 +39,20 @@ platform:
       - <wildcard_ipv4>
       - <wildcard_ipv6>
 ----
+
+ifdef::vSphere[]
+[IMPORTANT]
+====
+You can configure dual-stack networking on single interface only.
+====
+
+[NOTE]
+====
+* In a vSphere cluster configured for dual-stack networking, the node custom resource object has only the IP address from the primary network listed in `Status.addresses` field.
+* In the pod that uses the host networking with dual-stack connectivity, the `Status.podIP` and `Status.podIPs` fields contain only the IP address from the primary network.
```

Comment:
Do we need to state where the `Status.podIP` and `Status.podIPs` fields are, or will folks know?

Reply:
It's roughly standard notation.  It's in the YAML or JSON.

```
oc get pod  -o jsonpath='{.status.podIP}'
```

Reply:
@mburke5678 It is a known field since other parameters related to the dual-stack networking refer to the same yaml